docs/plans/apdd-governance-kit/explorer.html

```
@@ -0,0 +1,560 @@
+    .grid3 { grid-template-columns: 1fr 1fr; }
+    .mods { grid-template-columns: 1fr 1fr; }
+  }
+  @media (prefers-reduced-motion: reduce) { * { transition: none !important; animation: none !important; } }
+</style>
+</head>
+<body>
+
+<div class="topbar">
+  <div class="wrap row">
+    <div class="brand"><span class="dot"></span> <b>APDD&nbsp;KIT</b> &nbsp;·&nbsp; governance evaluation</div>
+    <button class="toggle" id="themeToggle" aria-label="Toggle color theme">◐ Theme</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)